pkgs/kde/frameworks/baloo/default.nix

```diff
@@ -6,5 +6,10 @@
 mkKdeDerivation {
   pname = "baloo";
 
+  # kde-systemd-start-condition is not part of baloo
```

Comment:
Of what is it part then? (Can't check on Mobile)
We should probably hardcode it to that is possible.

Reply:
`plasma-workspace`, and we can't hardcode because there is a cyclic dependency.

Reply:
Can we split the outputs for baloo to avoid it maybe?

Reply:
I don't see how split outputs would help with a cyclic dependency.

Reply:
We could have the dependency in the `bin` output of baloo, and have plasma-workspace only depend on `out`.

Reply:
Wouldn't that still lead to infinite recursion at eval time?

Reply:
No.

Reply:
Actually it does.

Reply:
Is the binary in PATH? forcing things to use binaries from /run/current-system is not that great but if there is no other option we can use that

Reply:
It should be on PATH but it's also a systemd unit so it doesn't respect global PATH necessarily.

Reply:
Wouldn't be the correct approach then to use it from path and explicitly add it to the systemd units PATH?

Reply:
FWIW, I just checked whether Baloo 5 is affected too. It is, and the workaround is pretty much the same: https://github.com/NixOS/nixpkgs/blob/3cbf4fdc09fa89ebce4bac238e23bb90928d8bee/pkgs/development/libraries/kde-frameworks/baloo.nix#L31-L34